bindings/kotlin/ldk-node-android/lib/src/androidTest/kotlin/org/lightningdevkit/ldknode/AndroidLibTest.kt

```diff
@@ -16,9 +16,9 @@ import androidx.test.ext.junit.runners.AndroidJUnit4
 @RunWith(AndroidJUnit4::class)
 class AndroidLibTest {
     @Test fun node_start_stop() {
-        val network: Network = "regtest"
-        assertEquals(network, "regtest")
+        val network = Network.REGTEST
 
+        val esplora_url = "http://127.0.0.1:3002"
```

Comment:
Changes look unrelated to the rest of the commit. Also, `esplora_url` is not used.